examples/gno.land/p/demo/mdui/mdui.gno

```diff
@@ -0,0 +1,391 @@
+func Link(text, href string) string {
+	return "[" + text + "](" + href + ")"
+}
+
+// Table generates a Markdown table
+func Table(headers []string, rows [][]string) string {
+	table := "| " + strings.Join(headers, " | ") + " |\n"
+	table += "|" + strings.Repeat("---|", len(headers)) + "\n"
+	for _, row := range rows {
+		table += "| " + strings.Join(row, " | ") + " |\n"
+	}
+	return table
+}
+
+// Bold formats text in bold
+func Bold(text string) string {
+	return "**" + text + "**"
+}
+
+// Italic formats text in italic
+func Italic(text string) string {
+	return "_" + text + "_"
+}
+
+// Strikethrough adds a strikethrough to the text
+func Strikethrough(text string) string {
+	return "~~" + text + "~~"
+}
+
+// Alert creates a Markdown-styled alert block with a specified type or custom prefix
+func Alert(content, alertType string) string {
+	var prefix string
+	switch alertType {
+	case "info":
+		prefix = "**ℹ️ Info:** " // Info icon
+	case "warning":
+		prefix = "**⚠️ Warning:** " // Warning icon
+	case "danger":
+		prefix = "**❌ Danger:** " // Danger icon
+	case "success":
+		prefix = "**✅ Success:** " // Success icon
+	default:
+		prefix = "**" + strings.Title(alertType) + ":** " // Custom prefix based on type
+	}
+	return "> " + prefix + content + "\n"
+}
+
+// Collapsible creates a collapsible section with a title and content
+func Collapsible(title, content string) string {
+	return "<details>\n<summary>" + title + "</summary>\n\n" + content + "\n\n</details>\n"
+}
+
+// Footnote generates a Markdown footnote
+func Footnote(label, text string) string {
+	return "[^" + label + "]: " + text + "\n"
+}
+
+// Badge generates a Markdown badge (often used in documentation)
+func Badge(label, color string) string {
+	return "![](https://img.shields.io/badge/" + strings.ReplaceAll(label, " ", "%20") + "-" + color + ")"
+}
+
+// TableOfContents generates a simple table of contents based on an array of headings
+func TableOfContents(headings []string) string {
+	toc := "## Table of Contents\n"
+	for i, heading := range headings {
+		toc += strconv.Itoa(i+1) + ". [" + heading + "](#" + strings.ToLower(strings.ReplaceAll(heading, " ", "-")) + ")\n"
+	}
+	return toc
+}
+
+// KeyboardShortcut formats a keyboard shortcut in Markdown using code block style
+func KeyboardShortcut(keys ...string) string {
+	return "`" + strings.Join(keys, " + ") + "`"
+}
+
+// Emoji adds an emoji to the content using a shortcode
+func Emoji(name string) string {
+	emojiMap := map[string]string{
+		"smile":     "😊",
+		"heart":     "❤️",
+		"thumbs_up": "👍",
+		"star":      "⭐",
+		"fire":      "🔥",
+		"check":     "✔️",
+		"cross":     "❌",
+		"warning":   "⚠️",
+		"info":      "ℹ️",
+		"sun":       "☀️",
+		"moon":      "🌙",
+		"rocket":    "🚀",
+		// Add additional emojis as needed
+	}
+	if emoji, exists := emojiMap[name]; exists {
+		return emoji
+	}
+	// If emoji is not found, return the name itself as a fallback
+	return ":" + name + ":"
+}
```

Comment:
I think we can drop this. It takes me the same amount of time to look up an emoji online or in your package; and it will need to be maintained.